examples/tutorial.ipynb

```diff
@@ -159,8 +159,6 @@
     "HAVE_PASQAL_ACCOUNT = False # If you have a PASQAL Cloud account, fill in the details and set this to `True`.\n",
```

Comment:
```suggestion
    "VALID_PASQAL_ACCOUNT = False # If you have a PASQAL Cloud account, fill in the details and set this to `True`.\n",
```

Reply:
Do you think it's really better?

Reply:
Well, what do _you_ think ? It is only a suggestion.

Reply:
I kinda prefer my version :sweat_smile: